docs/builders/ephemeris/index.md

```diff
@@ -10,6 +10,7 @@ nav_order: 2
 
 Pages that deal with different types of Ephemeres that are supported:
 
-- [Using JPL Horizons Ephemeris for Solar System Objects](horizons)
+- [Using the GUI in OpenSpace to download Horizons data](horizons-gui)
```

Comment:
Should be big Ds:    "Using the GUI in OpenSpace to Download Horizons Data" to match the format of the other page titles